### ISSUE TITLE
Consortium member list in published nutraceuticals manuscript

### ISSUE THREAD

**Issue by agitter** [greenelab/covid19-review#976]
The first manuscript (nutraceuticals) has been published in _mSystems_: https://doi.org/10.1128/mSystems.00122-21

This is an exciting milestone.  However, I noticed that the consortium members are listed in an odd way in the HTML version.  All members are listed twice
![image](https://user-images.githubusercontent.com/3532898/126880850-f1050013-a9f5-4fae-bc43-cd86ff707afb.png)

It's correct in the PDF version and in [PubMed](https://pubmed.ncbi.nlm.nih.gov/33947804/).

**Comment by cbrueffer** [comment]
That's fantastic; congrats everyone, particularly Ronan and Halie!

**Comment by rando2** [comment]
Very odd, thank you for catching this! @cgreene do you have any thoughts on whether this is something _mSystems_ would be able to fix (or whom to contact there?)

**Comment by agitter** [comment]
@rando2 when I had similar issues in the past at other journals, I started with the generic journal staff email (msystems@asmusa.org).  They usually have a production team that can handle this type of problem.  When the error affects the HTML display only and not the PDF, I ask to have the HTML fixed without a formal correction to the article.

**Comment by rando2** [comment]
I believe this issue is resolved now!
![Screen Shot 2021-08-04 at 9 53 32 AM](https://user-images.githubusercontent.com/8975841/128193165-3c9503f2-b8df-4473-a110-bbd1457eda80.png)

**Comment by cbrueffer** [comment]
Not quite, there are a few people listed twice: you @rando2, Casey, Ronan (maybe I've missed someone). It seems they just expanded the Review Consortium, so people listed as original authors are now listed twice.

**Comment by rando2** [comment]
> Not quite, there are a few people listed twice: you @rando2, Casey, Ronan (maybe I've missed someone). It seems they just expanded the Review Consortium, so people listed as original authors are now listed twice.

I think these are OK because I think what ended up happening is that those 3 are listed as masthead authors (of this particular paper) and then also as part of the consortium. It is definitely weird to have authors listed twice, but I think it would be weirder to remove us from the consortium. Typically I haven't seen the consortium authors enumerated so high up in the paper (I feel like it's usually towards the bottom of the first page or at the end) but given that that's how they're doing it, I think this works? It will be kind of hilarious for the bigger papers where like 25/50 consortium authors are also masthead authors, though!

**Comment by agitter** [comment]
Not listing all consortium members twice is an improvement.  It still seems like they are treating the consortium name and all its members as a single "author", which is unusual and not compatible with Manubot citation.  The updated HTML version doesn't bother me as much.  However, we won't be able to use Manubot to cite this article by DOI correctly if they list all consortium members as a single author.
![image](https://user-images.githubusercontent.com/3532898/128611778-77a655e0-7563-4a45-b6de-b293ae01d833.png)

**Comment by rando2** [comment]
I can check with Casey about this next week-- he's out of town this week but I usually meet with him on Wednesdays. He's an editor there so might have a little more insight about what's going on and whether they'd be able to fix it!

**Comment by rando2** [comment]
@agitter Would the preferred solution be to just list "COVID-19 Consortium" in the metadata and then just have a static list somewhere else? I feel like that's usually how I see it done. Casey had a suggestion about who to email, but I just want to be sure I'm asking for the right thing!

**Comment by agitter** [comment]
> Would the preferred solution be to just list "COVID-19 Consortium" in the metadata and then just have a static list somewhere else?

Yes, that is what I'm used to.  I checked an arbitrary recent DREAM paper and that's what they did.  The [journal](https://doi.org/10.1038/s41746-021-00414-7) lists the consortium as a "literal" author (no first, last name) in the author list and then the members in text later.
![image](https://user-images.githubusercontent.com/3532898/130528971-0b1f146d-26f9-4d24-828f-f208c97002f3.png)
![image](https://user-images.githubusercontent.com/3532898/130528999-0499dd45-c9a7-4057-b3af-b1c3437dd07e.png)

When it is set up that way, the publisher can also pass the consortium members to [PubMed](https://pubmed.ncbi.nlm.nih.gov/33742069/)
![image](https://user-images.githubusercontent.com/3532898/130529061-348cc84f-4d1a-462d-9555-19b979857a33.png)

[Another example](https://pubmed.ncbi.nlm.nih.gov/32302568/) is formatted similarly.

I realize this is nit-picky, but it seemed worthwhile to refine the consortium display on this first paper before the rest are published.

**Comment by agitter** [comment]
The Crossref data must have propagated.  Now the author list is much better, though the consortium author shows up as the last author.  Here's the latest Manubot citation:

 **Dietary Supplements and Nutraceuticals under Investigation for COVID-19 Prevention and Treatment**
Ronan Lordan, Halie M Rando, Casey S Greene, COVID-19 Review Consortium *mSystems* (2021-06-29) <https://doi.org/gj9qvs>
DOI: [10.1128/msystems.00122-21](https://doi.org/10.1128/msystems.00122-21) · PMID: [33947804](https://www.ncbi.nlm.nih.gov/pubmed/33947804) · PMCID: [PMC8269209](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8269209)

**Comment by agitter** [comment]
The journal contact said that they cannot control the author order in DataCite.  It comes from Crossref.

Our Manubot citation options are:
`$ manubot cite --format markdown doi:10.1128/msystems.00122-21`
1. **Dietary Supplements and Nutraceuticals under Investigation for COVID-19 Prevention and Treatment**
Ronan Lordan, Halie M Rando, Casey S Greene, COVID-19 Review Consortium *mSystems* (2021-06-29) <https://doi.org/gj9qvs>
DOI: [10.1128/msystems.00122-21](https://doi.org/10.1128/msystems.00122-21) · PMID: [33947804](https://www.ncbi.nlm.nih.gov/pubmed/33947804) · PMCID: [PMC8269209](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8269209)

`$ manubot cite --format markdown pubmed:33947804`
1. **Dietary Supplements and Nutraceuticals under Investigation for COVID-19 Prevention and Treatment.**
Ronan Lordan, Halie M Rando, Casey S Greene *mSystems* (2021-05-04) <https://www.ncbi.nlm.nih.gov/pubmed/33947804>
DOI: [10.1128/msystems.00122-21](https://doi.org/10.1128/msystems.00122-21) · PMID: [33947804](https://www.ncbi.nlm.nih.gov/pubmed/33947804) · PMCID: [PMC8269209](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8269209)

`$ manubot cite --format markdown https://doi.org/10.1128/mSystems.00122-21`
1. **Dietary Supplements and Nutraceuticals under Investigation for COVID-19 Prevention and Treatment**
mSystems <https://journals.asm.org/doi/abs/10.1128/mSystems.00122-21>

`$ manubot cite --format markdown https://pubmed.ncbi.nlm.nih.gov/33947804/`
1. **Dietary Supplements and Nutraceuticals under Investigation for COVID-19 Prevention and Treatment**
Ronan Lordan, Halie M Rando, COVID-19 Review Consortium, Casey S Greene *mSystems* (2021-05-04) <https://pubmed.ncbi.nlm.nih.gov/33947804/>
DOI: [10.1128/msystems.00122-21](https://doi.org/10.1128/mSystems.00122-21)

`$ manubot cite --format markdown https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8269209/`
1. **Dietary Supplements and Nutraceuticals under Investigation for COVID-19 Prevention and Treatment**
Ronan Lordan, Halie M Rando, Casey S Greene *mSystems* <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8269209/>
DOI: [10.1128/msystems.00122-21](https://doi.org/10.1128/mSystems.00122-21)

The URL citation `https://pubmed.ncbi.nlm.nih.gov/33947804/` gets the metadata right, so I'll switch to that one for our manuscript.  We can keep this issue open until the journal corrects their HTML.

**Comment by rando2** [comment]
Thank you @agitter! I'm sorry this has been such a headache!
